### ISSUE TITLE
No module named 'mmcv._ext'

### ISSUE THREAD

**Issue by AshishGupta1992** [DevashishPrasad/CascadeTabNet#82]
I am trying to run the main.py file in my windows and facing the attached error. I have installed all the mentioned libraries. 
![Issue](https://user-images.githubusercontent.com/5601603/93168842-710f8980-f741-11ea-8168-7a5bd94f6553.PNG)

Please let me know if someone has faced this issue and how can i resolve it.

**Comment by Atul997** [comment]
Install the 0.4.3 version of mmcv.
`pip install mmcv==0.4.3`

**Comment by luke4u** [comment]
Hi @Atul997 , I got the same error from` mmcv =  1.0.5`. But if install 0.4.3, mmdet requires to have 1.0.5. So wondering which mmdet version do you need? I have `mmdet ==2.3.0`

**Comment by Atul997** [comment]
Try it with mmdet==1.2.0!

**Comment by luke4u** [comment]
@Atul997 , there seems no distribution for `mmdet==1.2.0`.  See this link https://pypi.org/project/mmdet/#history

**Comment by EHadoux** [comment]
Yep, you need to clone the Github repo and compile the project. Or at lease that's what is done in the colab notebook.
